modules/widgets/kickoff.nix

```
@@ -83,6 +83,16 @@ in
         };
       showActionButtonCaptions = mkBoolOption "Whether to display captions ('shut down', 'log out', etc.) for the footer action buttons";
       pin = mkBoolOption "Whether the popup should remain open when another window is activated.";
+      popupHeight = mkOption {
+        type = with types; ints.positive;
+        default = null;
+        example = 500;
+      };
+      popupWidth = mkOption {
+        type = with types; ints.positive;
+        default = null;
+        example = 700;
+      };
       settings = mkOption {
         type = with types; nullOr (attrsOf (attrsOf (either (oneOf [ bool float int str ]) (listOf (oneOf [ bool float int str ])))));
         default = null;
```

Settings options should be updated so they can also use top level widgets (for every single widget).

```suggestion
      settings = mkOption {
        type = with types; nullOr (attrsOf (either
          (attrsOf (either (oneOf [ bool float int str ]) (listOf (oneOf [ bool float int str ]))))
          (either (oneOf [ bool float int str ]) (listOf (oneOf [ bool float int str ])))
        ));
        default = null;
```